package.json

```diff
@@ -117,6 +117,7 @@
     "typedoc-plugin-markdown": "^3.13.6"
   },
   "resolutions": {
+    "**/request/qs": "^6.5.3",
```

Comment:
What's the `**/` prefix notation here?

Reply:
From https://github.com/yarnpkg/rfcs/blob/master/implemented/0000-selective-versions-resolutions.md:

> **/a denotes all the nested dependencies a of the project.

To specify all transitive dependencies of `qs`.